.github/workflows/ci-performance-metrics.yml

```
@@ -0,0 +1,533 @@
+
+      - name: 📤 Download Performance Reports
+        uses: actions/download-artifact@v4
+        continue-on-error: true
+        with:
+          name: ci-performance-reports-${{ github.run_number }}
+          path: ./performance-reports/
+
+      - name: 🔍 Detect Performance Regressions
+        run: |
+          echo "🔍 Checking for performance regressions..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)